.github/workflows/gradle.yml

```diff
@@ -107,10 +107,8 @@ jobs:
       - name: Check PAT rate limits
         run: |
           ./tools/bin/find_non_rate_limited_PAT \
-            ${{ secrets.AIRBYTEIO_PAT }} \
-            ${{ secrets.OSS_BUILD_RUNNER_GITHUB_PAT }} \
-            ${{ secrets.SUPERTOPHER_PAT }} \
-            ${{ secrets.DAVINCHIA_PAT }}
+            ${{ secrets.GH_PAT_BUILD_RUNNER_OSS }} \
+            ${{ secrets.GH_PAT_BUILD_RUNNER_BACKUP }}
```

Comment:
The root cause why we hitted `no more requests` in this action is that it always used `AIRBYTEIO_PAT` as first one whilst this pat was being consumed by other workflows. Since then, now we have a division of PATs per role and repo, so there should not be any `no more requests` errors anymore.